src/screens/Dashboard.js

```diff
@@ -50,15 +51,13 @@ class Dashboard extends React.Component {
   }
 
   render() {
+    const blocks = this.props.data.length > 0 ? this.props.data[this.props.data.length - 1].blocks : '';
+    const transactions = this.props.data.length > 0 ? this.props.data[this.props.data.length - 1].transactions : '';
     return (
       <div className="content-wrapper">
-        <LineChartRealTime data={this.props.data} getX={this.getXData} getY={this.getYTxRate} unit="tx/s" title={["Tx Rate"]}/>
-        <LineChartRealTime data={this.props.data} getX={this.getXData} getY={this.getYTx} title={["Transactions"]} />
-        <LineChartRealTime data={this.props.data} getX={this.getXData} getY={this.getYBlock} title={["Blocks"]}/>
```

Comment:
Both the number of blocks and the number of transactions are useful pieces of information. I think we should either keep the chart (over time) or show the latest number.

Reply:
I've deleted the chart but added the information. Check https://github.com/HathorNetwork/hathor-explorer/pull/11/files#diff-fb9617d2e3aab8062b6397c120d67398R58 also see how it is in the screen shot

Reply:
Seems good!

Reply:
Thanks!